Snakefile

```diff
@@ -62,8 +68,13 @@ elif config["run"]["pipeline"] == "wgs":
     include: base + "sv.smk"
     include: base + "svreport.smk"
     include: base + "str.smk"
+    include: base + "mito_variants.smk"
 elif config["run"]["pipeline"] == "annot":
     base = "rules/"
+elif config["run"]["pipeline"] == "mity":
+    base = "rules/"
+    include: base + "mito_variants.smk"
```

Comment:
This also needs to include `base + "mapping.smk"` if the intent is to run the genome from scratch

Reply:
you're right, I'll add that in!